recipes/agent.rb

```diff
@@ -51,6 +53,13 @@
   chef_environment: node.chef_environment
 }
 
+# set lumberjack key locations and perms
+node.default['lumberjack']['ssl_dir'] = node['logstash']['instance_default']['basedir']
+node.default['lumberjack']['ssl_key_path'] = "#{node['lumberjack']['ssl_dir']}/#{node['lumberjack']['ssl key']}"
+node.default['lumberjack']['ssl_cert_path'] = "#{node['lumberjack']['ssl_dir']}/#{node['lumberjack']['ssl certificate']}"
+node.default['lumberjack']['user'] = node['logstash']['instance_default']['user']
+node.default['lumberjack']['group'] = node['logstash']['instance_default']['group']
+
```

Comment:
If these were in `attributes/` instead of a recipe, then not only will they need to use `override`, but if they want to override one value, they'd actually end up needing to override all five, e.g. `override['lumberjack']['ssl_dir'] = '/foo'` wouldn't actually change `node['lumberjack']['ssl_key_path']` or `node['lumberjack']['ssl_cert_path']` in an attribute file.

But since these are in a recipe, I guess it does end up working out okay.